astropy/table/tests/test_init_table.py

```diff
@@ -570,3 +571,17 @@ def test_init_data_type_not_allowed_to_init_table():
     with pytest.raises(ValueError,
                        match="Data type <class 'str'> not allowed to init Table"):
         Table('hello')
+
+
+def test_init_Table_from_list_of_quantity():
```

Comment:
How about running the same test on `QTable` as well (which already worked, but might as well ensure it doesn't regress!).

Reply:
I think that case is covered here: https://github.com/astropy/astropy/blob/11cbb7612dd21ad986e5c7642a0f22d7885ef264/astropy/table/tests/test_table.py#L2775

Remember that `Table(rows=rows)` flips the rows to a list of columns with `data = list(zip(*rows))`.

Reply:
Yes, that indeed covers it!